eva-server/src/test/resources/test-data/variants.json

```diff
@@ -221,6 +221,24 @@
           "name":"X:g.1000014G>A"
         }
       ]
+    },
+    {
+      "type":"SNV",
```

Comment:
SNV means "single nucleotide variation", which means "only one nucleotide was changed". These variants are one deletion and one insertion. The type should be "DEL" and "INS" respectively.  see https://github.com/EBIvariation/variation-commons/blob/master/variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantType.java